lib/logstash/outputs/rabbitmq.rb

```diff
@@ -15,6 +18,8 @@ class RabbitMQ < LogStash::Outputs::Base
       include LogStash::PluginMixins::RabbitMQConnection
 
       config_name "rabbitmq"
+      
+      default :codec, "json"
```

Comment:
That allows documentation to be associated with the setting, which is probably a good thing here.